package.json

```diff
@@ -2,19 +2,21 @@
   "name": "ping-centre",
   "version": "1.0.0",
   "description": "A client for easily collecting events and metrics.",
-  "main": "ping-centre.js",
+  "main": "src/ping-centre.js",
```

Comment:
Should this be **src/ping-centre.js** or the webpack-ified **dist/ping-centre.min.js**?